src/app/pages/PerpetualPage/contexts/PerpetualQueriesContext.tsx

```diff
@@ -164,6 +177,26 @@ export const PerpetualQueriesContextProvider: React.FC<PerpetualQueriesContextPr
     [refetchContract, refetchUser],
   );
 
+  const refetchBalance = useMemo(
+    () =>
+      debounce(
+        () =>
+          bridgeNetwork
+            .call(
+              Chain.BSC,
+              getContract('PERPETUALS_token').address,
+              marginTokenAbi,
+              'balanceOf',
+              [account],
+            )
+            .then(result => result && setAvailableBalance(String(result)))
+            .catch(console.error),
+        500,
+        { leading: false, trailing: true, maxWait: 500 },
```

Comment:
`leading: false` and `traling: true` are the default values (according to the [documentation](https://lodash.com/docs/4.17.15#debounce)) so there is no need to specify them.